### ISSUE TITLE
pre-populata data in SSR

### ISSUE THREAD

**Issue by cometta** [prescottprue/react-redux-firebase#209]
Is firebase request can only be done on client side instead of server side?, my ` "data": {},` is empty and thus request for data is only done by client side.

> window.__INITIAL_STATE__ = {
>   "firebase": {
>     "requesting": {
>       "todos": true
>     },
>     "requested": {
>       "todos": false
>     },
>     "timestamps": {
>       "todos": 1501123754524
>     },
>     "data": {},
>     "ordered": {},
>     "auth": {
>       "isLoaded": true,
>       "isEmpty": true
>     },
>     "profile": {
>       "isLoaded": true,
>       "isEmpty": true
>     },
>     "listeners": {
>       "byId": {
>         "value:/todos": {
>           "id": "value:/todos",
>           "path": "/todos"
>         }
>       },
>       "allIds": ["value:/todos"]
>     },
>     "isInitializing": false,
>     "errors": []
>   }
> };

**Comment by prescottprue** [comment]
Not sure I understand what you are asking. Can you possibly provide a repository or example where the issue can be replicated?

**Comment by cometta** [comment]
@prescottprue , i dont have repo. Let me rephrase my question.  Is it possible to retrieve data from firebase at serverside?

**Comment by prescottprue** [comment]
@cometta Yes is the simple answer, but without knowing more about what you are trying to do, it is hard to try to give an example.

**Comment by cometta** [comment]
@prescottprue , take the todos for example, instead of retrieving the list from client side, i would like the SSR to retrieve those information and render as html so that  client side does not require to make additional request to retrieve the todo list
